src/cincinnati/mod.rs

```diff
@@ -232,7 +232,13 @@ fn find_update(
         .find(|(_, node)| is_same_checksum(node, &booted_depl.checksum))
     {
         Some(current) => current,
-        None => return Ok(None),
+        None => {
+            log::warn!(
+                "booted deployment {} not found in the update graph",
+                &booted_depl.payload
```

Comment:
At this point in time (this commit) I don't think `rpm_ostree::Release` has been updated from `pub checksum: String,` to `pub payload: Payload,`.

Maybe this commit should go last.

Reply:
good catch ! I'll reorder